posts/2024-07-10-rln-v2-stateless-light-clients.mdx

```diff
@@ -1,68 +1,72 @@
 ---
 layout: post
-name: 'Improved rate limiting and stateless light clients in The Waku Network'
-title: 'Improved rate limiting and stateless light clients in The Waku Network'
+name: 'Improved rate limiting and stateless light clients in the Waku Network'
```

Comment:
We use capital letter here since we refer to TWN (The Waku Network) everywhere

Reply:
Gotcha, changed this back to capital 'The'. Was referencing this blog post I remember editing: https://blog.waku.org/2024-waku-network-tech-overview
I can change it here too though if teh agreed standard is 'The'